app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split(" ")
+    cp = command[0]
+    file_from = command[1]
+    file_to = command[2]
+    if cp == "cp" and file_from != file_to:
+        with open(file_from, "r") as f:
+            content = f.read()
+        with open(file_to, "w") as f:
+            f.write(content)
```

Comment:
It is bad practice, but i think it is not mistake. 
You can use two contexts managers together, it will be more easy to read and smaller code
![image](https://github.com/user-attachments/assets/90cf0648-f776-48c5-826b-6ed0e8af149a)